bundle/language_en.properties

```diff
@@ -810,6 +812,7 @@ angal.medicalstock.lot.col
 angal.medicalstock.lotduedate                                                                          = Lot Due Date
 angal.medicalstock.lotid                                                                               = Lot id
 angal.medicalstock.lotpreparationdate                                                                  = Lot Preparation Date
+angal.medicalstock.lastmovementnotfound.msg                                                            = Last movement not found.
```

Comment:
I think this is not used anymore, correct?

Reply:
yes